hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ConfigurationService.java

```diff
@@ -57,6 +59,23 @@
      */
     void broadcastConfig(IdentifiedDataSerializable config);
 
+    /**
+     * Reloads the configuration with the given configuration. Reloading means
+     * dynamically changing all the differences dynamically changeable.
+     *
+     * @param newConfig config to find any new dynamically changeable sub configs
+     * @return reload result which includes added and ignored configurations
+     */
+    ReloadResult reload(Config newConfig);
```

Comment:
It is updating with given config, I find the reload name here confusing,

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/79815902c5fdbee436acacbeae16a04cbad5135e